ui/StatusQ/src/themepalette.cpp

```diff
@@ -0,0 +1,459 @@
+#include "StatusQ/themepalette.h"
+#include "StatusQ/statuscolors.h"
+#include <QtQml/qqml.h>
+
+ThemePalette::ThemePalette(QObject* parent)
+    : QObject(parent)
+{
+    black = QColor(0,0,0);
+    white = QColor(255,255,255);
+    transparent = QColor(0,0,0,0);
+}
+
+QColor ThemePalette::hoverColor(const QColor& normalColor) const
+{
+    if (name == QLatin1String("light"))
+        return QColor(normalColor).darker(120);
+    return QColor(normalColor).lighter(120);
```

Comment:
```suggestion
    return normalColor.lighter(120);
```

No need to create a new color